app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -327,6 +338,18 @@ const SendBitcoinConfirmationScreen: React.FC<Props> = ({ route }) => {
     DisplayCurrency,
   )
 
+  if (paymentDetail.settlementAmount.amount < 546) {
+    feeDisplayText = LL.SendBitcoinConfirmationScreen.dustError()
+  }
+
+  if (paymentDetail.settlementAmount.amount < 546) {
+    feeDisplayText = LL.SendBitcoinConfirmationScreen.dustError()
+  }
+
+  if (paymentDetail.settlementAmount.amount < 546) {
+    feeDisplayText = LL.SendBitcoinConfirmationScreen.dustError()
+  }
+
```

Comment:
why is this copy/paste 3 times?

Reply:
I think this occurred when I was resolving merge conflicts, I will change it